services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/attachment/application/ApplicationAttachmentServiceITest.kt

```diff
@@ -103,30 +107,30 @@ class ApplicationAttachmentServiceITest(
     @Nested
     inner class GetMetadataList {
         @Test
-        fun `returns empty when application doesn't exist`() {
+        fun `Returns empty when application doesn't exist`() {
             val applicationId = 698432169874L
 
-            val result = applicationAttachmentService.getMetadataList(applicationId)
+            val result = attachmentService.getMetadataList(applicationId)
 
             assertThat(result).isEmpty()
         }
 
         @Test
-        fun `return empty when application doesn't have attachments`() {
+        fun `Return empty when application doesn't have attachments`() {
```

Comment:
"Returns", "Return" "Returns". Just a small detail but since you changed r --> R I noticed this.

Reply:
Typo fixed.